test/test_images/helloserviceaccount/helloserviceaccount.go

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
Is this image even needed given we can achieve the same using hellovolume and validation in the test itself?